### ISSUE TITLE
"rhoas status kafka" doesn't return Kafka cluster status

### ISSUE THREAD

**Issue by sutaakar** [redhat-developer/app-services-cli#1425]
## Description

**Version**: 0.38.2

Command `rhoas status kafka` doesn't find any Kafka instance even though I have configured the existing Kafka cluster using `rhoas kafka use --name <kafka name>`.

### Steps to reproduce

Create new Kafka cluster, wait until it is in ready status.
Login using `rhoas login`
Set created Kafka cluster as current context using `rhoas kafka use --name <created Kafka cluster name>`
Run `rhoas status kafka` - > the command returns `No services are currently being used.` instead of returning Kafka cluster status.

### Expected vs actual behaviour
I expect that  `rhoas status kafka` returns informations about Kafka cluster configured as current context, but command doesn't find any Kafka cluster.

### Logs
```
[ksuta@fedora ~]$ rhoas status kafka

No services are currently being used. To set a service in context, run "rhoas [service] use [args]".
```

The command works properly in version 0.37.0, but failing on 0.38.1 or 0.38.2.

**Comment by wtrocki** [comment]
@sutaakar Thank you for logging. We will investigate and resolve this issue as soon as possible.

**Comment by rkpattnaik780** [comment]
Thanks a lot for reporting the issue @sutaakar .
A fix has been raised and a patch release will be made soon.

**Comment by rkpattnaik780** [comment]
Fixed in [0.38.3](https://github.com/redhat-developer/app-services-cli/releases/tag/v0.38.3).
